bluesky/plans.py

```diff
@@ -81,6 +81,7 @@ def __iter__(self):
         for token in tokens:
             yield Msg('unsubscribe', token)
         yield from self._post()
+        yield Msg('checkpoint')
```

Comment:
Do we also have an edge case with `chekpoint->open_run->pause` where we will double emit the run start?

Do we need to add a `clear_checkpoint` message?